.rubocop.yml

```diff
@@ -18,7 +18,9 @@ Style/PercentLiteralDelimiters:
     '%W': '[]'
 Style/StringLiterals:
   Enabled: false
-Style/TrailingComma:
+Style/TrailingCommaInLiteral:
+  Enabled: false
+Style/TrailingCommaInArguments:
```

Comment:
Ah, sorry, missed this before merging. The file isn't alphabetized
currently – I'll leave that for the next changeset.

On Mon, Apr 18, 2016 at 1:15 PM, Ashley Baldwin-Hunter <
notifications@github.com> wrote:

> In .rubocop.yml
> <https://github.com/codeclimate/codeclimate-rubocop/pull/64#discussion_r60096976>
> :
>
> > @@ -18,7 +18,9 @@ Style/PercentLiteralDelimiters:
> >      '%W': '[]'
> >  Style/StringLiterals:
> >    Enabled: false
> > -Style/TrailingComma:
> > +Style/TrailingCommaInLiteral:
> > +  Enabled: false
> > +Style/TrailingCommaInArguments:
>
> 💄 what do you think about alphabetizing these?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/codeclimate/codeclimate-rubocop/pull/64/files/ceaa4cc9a8f1dcb336b1b20237a4a149a7b9ab5d#r60096976>
>